server/db/utils/transformCallback.js

```diff
@@ -14,7 +14,7 @@ export const transformCallback = (row, draft = false, assocPublishedDraft = fals
 
   const rowUpdated = A.pipe(
     // Assoc published and draft properties based on props
-    (rowCurrent) => (assocPublishedDraft ? _assocPublishedDraft(rowCurrent) : rowCurrent),
+    (rowCurrent) => (backup || assocPublishedDraft ? _assocPublishedDraft(rowCurrent) : rowCurrent),
```

Comment:
is this included in arena-server ?

Reply:
uhm, no, also in arena-server I should include (maybe) a change to the Job class

Reply:
why a change to the job class?

Reply:
to set inner job parameters/context into the parent job context

Reply:
can you open a PR then ?

Reply:
it's not needed anymore for Job, I will open a PR for transformCallback only